website/pages/docs/job-specification/lifecycle.mdx

```diff
@@ -11,58 +11,130 @@ description: |-
 
 <Placement groups={['job', 'group', 'task', 'lifecycle']} />
 
-The `lifecycle` stanza is used to express task dependencies in Nomad and
-configures when a task is run within the lifecycle of a task group.
+The `lifecycle` stanza is used to express task dependencies in Nomad by
+configuring when a task is run within the lifecycle of a task group.
 
 Main tasks are tasks that do not have a `lifecycle` stanza. Lifecycle task hooks
-are run in relation to the main tasks. Tasks can be run as Prestart Hooks, which
-ensures the prestart task is run before the main tasks are run.
+specify when other tasks are run in relation to the main tasks.
+There are three different lifecycle hooks, indicating when a task is started:
+* prestart tasks are immediately
+* poststart tasks are started after the main tasks are running
+* poststop tasks are started after the main tasks are dead
 
-Tasks can be run with an additional parameter to indicate that they are
-sidecars, which ensures that they are running over the duration of the whole
-task group. This will allow you to run a long-lived task in a task group for a
-batch job. The absence of the sidecar flag indicates that the task is ephemeral
-and the task will not be restarted if it completes successfully. This allows you
-to run an ephemeral prestart task in a task group for a service job, which can
-serve as initialization that occurs before the main services are started.
+Tasks can be run with an additional parameter indicating whether they are ephemeral
+tasks or "sidecar" tasks that are expected to run for the duration of the main tasks.
+The absence of the sidecar flag indicates that the task is ephemeral
+and should not be restarted if it completes successfully.
 
 Learn more about Nomad's task dependencies on the [HashiCorp Learn website][learn-taskdeps].
 
+## `lifecycle` Parameters
+
+- `hook` `(string: <required>)` - Specifies when a task should be run within
+  the lifecycle of a group. The following hooks are available:
+  - `prestart` - Will be started immediately. The main tasks will not start until
+    all `prestart` tasks with `sidecar == true` have completed successfully.
```

Comment:
Although `==` is "technically correct" for equality, I think we usually just use the exact syntax in the docs

```suggestion
    all `prestart` tasks with `sidecar = true` have completed successfully.
```

Reply:
thanks, i noticed that as well and missed this one!